tests/acceptance/features/webUIPreview/mediaPreview.feature

```diff
@@ -3,24 +3,6 @@ Feature: display image in preview app on the webUI
   Background:
     Given user "Alice" has been created with default attributes and without skeleton files in the server
 
-  @ocisSmokeTest
-  Scenario Outline: preview of image files with preview app is possible
-    Given user "Alice" has uploaded file "<image-file>" to "<image-file>" in the server
-    And user "Alice" has logged in using the webUI
-    When the user views the file "<image-file>" in the preview app using the webUI
-    Then the file "<image-file>" should be displayed in the preview app webUI
-    Examples:
-      | image-file      |
-      | testavatar.jpg  |
-      | testavatar.png  |
-      | testavatar.jpeg |
-
-  Scenario: preview of video with preview app is possible
-    Given user "Alice" has uploaded file "test_video.mp4" to "test_video.mp4" in the server
-    And user "Alice" has logged in using the webUI
-    When the user views the file "test_video.mp4" in the preview app using the webUI
-    Then the file "test_video.mp4" should be displayed in the preview app webUI
-
 
   Scenario: video playback in public share
```

Comment:
can we add this case to tests/e2e/cucumber/features/shares/link.feature and delete this test?